src/tasks/leveling.ts

```diff
@@ -990,7 +1050,7 @@ export const LevelingQuest: Quest = {
         !have($item`backup camera`) ||
         !freeFightMonsters.includes(get("lastCopyableMonster") ?? $monster.none) ||
         get("_backUpUses") >= 11 - clamp(get("instant_saveBackups", 0), 0, 11) ||
-        myBasestat($stat`Mysticality`) >= 190, // no longer need to back up Witchess Kings
+        myBasestat(myPrimestat()) >= 190, // no longer need to back up Witchess Kings
```

Comment:
All of these `myBasestat(myPrimestat())` can be changed to `myBasestat(mainStat)`